src/js/modules/sections/components/SectionPage.js

```diff
@@ -3,26 +3,121 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 import { Link } from 'react-router-dom';
 import injectSheet from 'react-jss';
-import sections from '../../sections';
+import { Grid, Row, Col } from 'react-bootstrap/lib/';
 
 import articles from '../../articles';
 
```

Comment:
For each class in the JSS, make sure its properties are sorted in alphabetical order.